config/releases.exs

```diff
@@ -12,30 +12,3 @@ config :boilerplate, Boilerplate.Repo,
   password: System.fetch_env!("DB_PASSWORD"),
   database: System.fetch_env!("DB_INSTANCE"),
   pool_size: 20
-
-config :logger,
-  level: :info,
-  backends: [:console, LoggerLagerBackend],
-  handle_otp_reports: false,
-  utc_log: true
-
-config :sentry,
-  filter: BoilerplateWeb.SentryEventFilter,
-  dsn: System.fetch_env!("SENTRY_DSN")
-
-config :lager,
```

Comment:
Does it work? I haven't seen it replaced in any configuration file.

Reply:
I've moved these configs to `/config/prod.exs`, should I put it somewhere else?

Reply:
No, I didn't found in the diff, but it looks like it's already there.